Server/controllers/inviteController.js

```
@@ -1,101 +1,111 @@
-      ? (error.method = "inviteVerifyController")
-      : null;
-    next(error);
-    return;
-  }
+    try {
+        const invite = await req.db.getInviteToken(req.body.token);
+        res
+            .status(200)
+            .json({status: "success", msg: "Invite verified", data: invite});
+    } catch (error) {
+        error.service === undefined ? (error.service = SERVICE_NAME) : null;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)